packages/forms/src/model.ts

```diff
@@ -2050,9 +2050,11 @@ export class FormArray extends AbstractControl {
   /**
    * Get the `AbstractControl` at the given `index` in the array.
    *
-   * @param index Index in the array to retrieve the control
+   * @param index Index in the array to retrieve the control. Negative integers count back from the
+   *     last item in the array.
    */
   at(index: number): AbstractControl {
+    index += index >= 0 ? 0 : this.length();
```

Comment:
It'd be great to move this logic to a separate function:
```suggestion
    index = adjustIndex(index, this);
```
where the `adjustIndex` function may look like this:
```
function adjustIndex(index: number, formArrayInstance: FormArray): number {
  return index >= 0 ? index : index + formArrayInstance.length();
}
```

Reply:
Good idea. Would you like it to be a class method or as a util function? If it's the latter, where should I place it (in this file, in a util module)?

Reply:
@amitbeck I was thinking to put it to a separate function, so that the function name can be minified (we don't do property renaming at this moment), but I guess the savings wouldn't be that big (since we'd also need to pass an argument in this case). Let's proceed with the class method.